src/core/coreconfig.h

```diff
@@ -66,6 +69,8 @@ namespace vnotex
         // Icon size of MainWindow tool bar.
         int m_toolBarIconSize = 16;
 
+        int m_minimize_to_system_tray = -1;
```

Comment:
How about `m_minimizeToSystemTray`? And add one comment about the value meaning.

Reply:
And I think it is better to put it in the `SessionConfig`.

Reply:
I have not learned the logic of SessionConfig, so I chose user configuration